scripts/dockerfiles/Dockerfile.buildCNIPlugins

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+FROM golang:1.7
```

Comment:
I have always built this with go1.7
Also [original cni-plugins](https://github.com/containernetworking/cni/blob/b69e640cc0e2b2be0e3544f28aff7466e64c8422/.travis.yml#L6) are also built using go >= 1.7.x